src/Illuminate/Database/Eloquent/Concerns/QueriesRelationships.php

```diff
@@ -951,6 +951,22 @@ public function withCount($relations)
         return $this->withAggregate(is_array($relations) ? $relations : func_get_args(), '*', 'count');
     }
 
+        /**
```

Comment:
You might reduce the indent here:
```suggestion
    /**
```